lambdas/jwt-signer/src/jwt-signer-handler.ts

```diff
@@ -0,0 +1,91 @@
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { createHash } from "crypto";
+import sigFormatter from "ecdsa-sig-formatter";
+import { fromEnv } from "@aws-sdk/credential-providers";
+import { SignerPayLoad } from "./signer-payload";
+import { SignageType } from "./signage-type";
+import { base64url } from "jose";
+import {
+  KMSClient,
+  MessageType,
+  SignCommand,
+  SigningAlgorithmSpec,
+} from "@aws-sdk/client-kms";
+import { Logger } from "@aws-lambda-powertools/logger";
+const logger = new Logger({ serviceName: "JwtSignerHandler" });
+
+export class JwtSignerHandler implements LambdaInterface {
+  constructor(private kmsClient: KMSClient) {}
+
+  public async handler(event: SignerPayLoad): Promise<string> {
+    const header = base64url.encode(event.header);
+    const payload = base64url.encode(event.claimsSet);
+
+    logger.info("header is: " + header);
+    logger.info("payload is: " + payload);
+
+    const response = await this.signWithKms(
+      header,
+      payload,
+      event.kid as string
+    );
+    const signature = sigFormatter.derToJose(
+      Buffer.from(response).toString("base64"),
+      "ES256"
+    );
+
+    const jwtString: string = `${header}.${payload}.${signature}`;
+
+    logger.info("Jwt String: " + jwtString);
```

Comment:
We probably dont want to be logging this string or the payload but could we add some logs indicating what is going on in the cri. i.e
encoding JWT details
Signing JWT
JWT Signed